.github/workflows/release-image.yml

```diff
@@ -0,0 +1,28 @@
+name: release image
+on:
+  push:
+    tags:
+      - 'image-v*' # push events for tags matching image-v for version (image-v1.0, etc)
+jobs:
+  image:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: set env
+        run: echo ::set-env name=RELEASE_VERSION::$(echo ${GITHUB_REF:17}) # refs/tags/image-v1.0.0 substring starting at 1.0.0
+      - name: setup buildx
+        uses: docker/setup-buildx-action@v1
+      - name: login to GitHub container registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.CR_PAT }}
+      - name: build and push
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          file: 'vmss-prototype/Dockerfile'

```

Comment:
I think we need a context of 'vmss-prototype'

Unsure but the way I read the docs we need to pick that (albeit it is strange you can't just give a context and default the docker file)

Reply:
Also, we may wish to change the name (in the future) of this directory given that the directory and the binary have the same name and could cause confusion for certain types of errors.

Reply:
https://github.com/docker/build-push-action#inputs

Reply:
added a path context